optimize/backend/pom.xml

```diff
@@ -398,6 +398,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
do you think it would be worth adding a requirement to @buccarel's single application issue that explicitly states that this dependency should be removed?